dataplane/vppagent/conf/vpp/startup.conf

```diff
@@ -71,7 +71,7 @@ cpu {
 }
 
 # dpdk {
-	## Change default settings for all intefaces
+	## Change default settings for all interfaces
```

Comment:
LOL... I appreciate your fastidious attention to detail :)

Reply:
Hahahahah :)) Why not?